backend/validators/serializers.py

```
@@ -57,6 +57,40 @@ def get_explorer_url(self, obj):
+        fields = [
+            'id',
+            'address',
+            'wallet_count',
+            'active_wallet_count',
+            'networks',
+            'created_at',
+            'updated_at',
+        ]
+        read_only_fields = fields
+
+    def _wallets(self, obj):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_wallets`

(ANN202)